opentelemetry-tracing/opentelemetry-java-tracing/okhttp-and-jedis/README.md

```diff
@@ -6,31 +6,32 @@ Instrumentation](https://docs.oracle.com/javase/8/docs/api/java/lang/instrument/
 interface and enables tracing of many popular libraries and frameworks without
 modification of the target application's source code.
 
+In order to run this example, you will need to run the opentelemetry collector locally and have it configured to 
+receive otlp spans and export data to Splunk's ingest endpoints. 
+
+TODO: provide a link to how to do set up the collector for this demo. Or, better yet, make it more self-contained.
```

Comment:
👍  💯 

We should use the [splunk collector distro](https://github.com/signalfx/splunk-otel-collector) if we ever make it self-contained.

Reply:
Should we create an issue for this? How should it be done? docker-compose? something else?

Reply:
Created https://github.com/signalfx/tracing-examples/issues/67 to track it